src/main/java/edu/greenblitz/tobyDetermined/commands/rotatingBelly/AlignObject.java

```diff
@@ -0,0 +1,11 @@
+package edu.greenblitz.tobyDetermined.commands.rotatingBelly;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class AlignObject extends SequentialCommandGroup {
+    public AlignObject(){
+        addCommands(new RotateOutOfDoor());
+        addCommands(new RotateInToDoor());
```

Comment:
presumably RotateInToDoor will take longer when not immediately preceded by RotateOutOfDoor and so this cant use the default time for rotate in door

Reply:
yes it can because it comes after what u said >:(